templates/shared/runtime/bin/imds

```diff
@@ -37,7 +37,7 @@ function imdscurl() {
     cat >&2 $OUTPUT_FILE
     return $CODE
   fi
-  printf "$(cat $OUTPUT_FILE)\n"
+  cat $OUTPUT_FILE
```

Comment:
Maintain the trailing newline:
```suggestion
  cat $OUTPUT_FILE
  printf “\n”
```

Reply:
oh hah i think the `\n` is already omit when shell subbing `$(cat ..)`
adding this line was breaking the tests